examples/particle/aws/kinesis/kinesis_ex.py

```diff
@@ -0,0 +1,35 @@
+from pcf.particle.aws.kinesis.kinesis import Kinesis
```

Comment:
naming convention for examples: "aws/use_[particle_name]/example_[particle_name].py"

Reply:
could you also include update in the example (changing one field and applying again.)